test-common/utils.h

```diff
@@ -285,6 +290,44 @@ gboolean eos_test_setup_flatpak_repo_with_preinstalled_apps_simple (GFile
                                                                     const gchar **preinstalled_flatpak_names,
                                                                     GError      **error);
 
+typedef enum _FlatpakInstallInfoType {
+  FLATPAK_INSTALL_INFO_TYPE_RUNTIME,
+  FLATPAK_INSTALL_INFO_TYPE_APP,
+  FLATPAK_INSTALL_INFO_TYPE_EXTENSION
+} FlatpakInstallInfoType;
+
+typedef struct _FlatpakInstallInfo {
```

Comment:
Nitpick: You don’t need the `_FlatpakInstallInfo` name. Just do
```
typedef struct
{
  …
} FlatpakInstallInfo;
```

Reply:
(Note: This looks like it still applies to a few of the other new structures in the branch.)